tensorpack/utils/export.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: UTF-8 -*-
+# File: exporter.py
+# Author: Patrick Wieschollek <mail@patwie.com>
+
+"""
+This simplifies the process of exporting a model for TensorFlow serving.
+
+"""
+
+import tensorflow as tf
+from tensorpack.utils import logger
+from tensorpack.tfutils import TowerContext
+from tensorpack.models import ModelDesc
+from tensorpack.tfutils import sessinit
+from tensorflow.python.saved_model import builder as saved_model_builder
+from tensorflow.python.saved_model import signature_constants, signature_def_utils, tag_constants, utils
```

Comment:
Since there are public APIs `tf.saved_model.builder` etc https://www.tensorflow.org/api_docs/python/tf/saved_model/builder/SavedModelBuilder, we can avoid these internal imports.